lib/validators/movie.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
```

Comment:
I know it doesn't really matter yet, but typically we use this convention for validators:

lib/validators/movie/create.js (this file)
lib/validators/movie/list.js (for listing movies, which you'll create later)